Package.swift

```diff
@@ -4,6 +4,31 @@
 
 import PackageDescription
 
+var tokamakDOMDependencies: [Target.Dependency] = [
+  "TokamakCore",
+  "TokamakStaticHTML",
+  .product(
+    name: "OpenCombineShim",
+    package: "OpenCombine"
+  ),
+  .product(
+    name: "JavaScriptKit",
+    package: "JavaScriptKit",
+    condition: .when(platforms: [.wasi])
+  ),
+  "OpenCombineJS",
+]
+
+#if compiler(>=5.5) && canImport(Concurrency)
```

Comment:
This conditional fails for me (Monterey/Xcode 13.1), while the following works.

```suggestion
#if compiler(>=5.5) && canImport(_Concurrency)
```

Reply:
I've replaced it with `#if compiler(>=5.5) && (canImport(Concurrency) || canImport(_Concurrency))`, which hopefully covers all possible cases. Does that fix the issue on your side?

Reply:
Yes, that works.